tests/www/apply/test_process.py

```diff
@@ -904,65 +904,69 @@ def test_refuse_session_prefix(self, client):
         job_application = JobApplicationFactory()
         employer = job_application.to_company.members.first()
         client.force_login(employer)
-        assert f"wizard_job_application_{job_application.pk}_refuse" not in client.session
 
-        url = reverse("apply:refuse", kwargs={"job_application_id": job_application.pk, "step": "reason"})
-        client.get(url)
+        url = reverse("apply:refuse", kwargs={"job_application_id": job_application.pk})
+        response = client.get(url)
 
-        assert client.session[f"wizard_job_application_{job_application.pk}_refuse"] == {
-            "step": None,
-            "step_data": {},
-            "step_files": {},
-            "extra_data": {},
+        [refuse_session_name] = [k for k in client.session.keys() if k not in KNOWN_SESSION_KEYS]
```

Comment:
il nous faudrait un helper pour remplacer ce 
`[k for k in client.session.keys() if k not in KNOWN_SESSION_KEYS]`